lib/ktl/reassignment_task.rb

```diff
@@ -21,7 +26,11 @@ def execute(dryrun = false)
           reassignment = @plan.generate
         end
         if reassignment.size > 0
-          @logger.info 'reassigning %d partitions' % reassignment.size
+          if (limit = @reassigner.limit)
+            @logger.info 'reassigning %d of %d partitions' % [limit, reassignment.size]
+          else
+            @logger.info 'reassigning %d partitions' % reassignment.size
+          end
```

Comment:
This logging also feels like it belongs in the reassigner, it assumes a lot about how the reassigners work.

I also prefer `sprintf`/`format` over `%` when there are multiple arguments, I think that `%` with a list argument looks unnecessarily weird.